script.js

```diff
@@ -1,21 +1,32 @@
 $(function () {
-
   //array to hold tarot cards
-  let cards = [  "death.png" , "devil.png" ,   "fool.png" , "hangedman.png" , "hermit.png" ,  "judgement.png" ,   "lovers.png" ,   "priestess.png" ,  "sun.png" ];
-
-//find random value for repeated card
+  let cards = [
+    "death.png",
+    "devil.png",
+    "fool.png",
+    "hangedman.png",
+    "hermit.png",
+    "judgement.png",
+    "lovers.png",
+    "priestess.png",
+    "sun.png"
+  ];
+
+  //find random value for repeated card
   let randomCard = cards[Math.floor(Math.random() * cards.length)];
-  console.log (randomCard)
+  console.log(randomCard);
+
+  //add random value to cards array
+  function addRandom() {
+    cards.push(randomCard);
+  }
 
-//add random value to cards array
-function addRandom () {
-  cards.push($(randomCard));
-}
+  addRandom();
```

Comment:
This was necessary to actually put the random double card back in the mix

Reply:
My code linter immediately informed me that ` addRandom` was defined but never used -- it is a good idea to develop in an IDE that can lint for you like that and help you catch little things.  

You can see that it works when called though, good work :-)